application/forms/Command/Object/RemoveAcknowledgementForm.php

```diff
@@ -7,13 +7,36 @@
 use Icinga\Module\Icingadb\Command\Object\RemoveAcknowledgementCommand;
 use Icinga\Module\Icingadb\Common\Auth;
 use Icinga\Module\Icingadb\Forms\Command\CommandForm;
+use Icinga\Module\Icingadb\Model\Host;
+use Icinga\Web\Notification;
 use ipl\Orm\Model;
 use ipl\Web\Widget\Icon;
 
 class RemoveAcknowledgementForm extends CommandForm
 {
     use Auth;
 
+    public function __construct()
+    {
+        $this->on(self::ON_SUCCESS, function () {
+            $countObjects = count($this->getObjects());
+            $message = sprintf(tp(
+                'Removed acknowledgment successfully',
+                'Removed acknowledgment problem from %d services successfully',
```

Comment:
What is a `acknowledgment problem`? Where does `problem` come from? This is of course an *acknowledged problem* that is getting un-acknowledged here, but it's the acknowledgement that's removed, not the problem.